connector.md

```diff
@@ -0,0 +1,42 @@
+# Connector  Library Sepcification
+
+## Unified Messaging Service
+
+The connector library exposes a service to bind to, allowing the distributor to send push related messages.
+
+### Manifest
+The library itself does not declare anything in the manifest. The developer integrating the library should add the following lines to the manifest Wher {PushReceiverService} is the service implementing the below specification.
+```
+<service android:name="{PushReceiverService}" android:exported="true">
+  <intent-filter>
+    <action android:name="org.unifiedpush.android.action.MESSAGING"/>
+  </intent-filter>
+</service>
+```
+
+### Incoming Handler
+
+The exposed service MUST handle 2 different types of messages:
```

Comment:
Did you forget TYPE_DITRIBUTOR_UNREGISTERED, or was it wanted?. The lib can work even if there isn't this type of message so I think it SHOULD handle this one